src/js/media-controller.js

```diff
@@ -487,10 +491,15 @@ class MediaController extends MediaContainer {
   attributeChangedCallback(attrName, oldValue, newValue) {
     if (attrName === 'nohotkeys') {
       if (newValue !== oldValue && newValue === '') {
+        if (this.hasAttribute('hotkeys')) {
+          console.warn('Both `hotkeys` and `nohotkeys` have been enabled. No hotkeys will trigger functionality.');
```

Comment:
`No hotkeys will trigger functionality` read a bit strange to me. 
maybe something like "all hotkeys are disabled / deactivated" would be more clear.